<launch>
  <node pkg="jsk_interactive_marker" type="interactive_marker_interface" name="jsk_interactive_marker_manipulation" output="screen">
    <!--
    <param name="base_frame" type="string" value="/odom"/>
    -->
    <param name="base_frame" type="string" value="/base_link"/>
    <param name="move_base_frame" type="string" value="/r_foot"/>
    <param name="move_menu" type="boolean" value="true"/>
    <param name="grasp_menu" type="boolean" value="true"/>
    <param name="stop_grasp_menu" type="boolean" value="true"/>
    <param name="harf_grasp_menu" type="boolean" value="true"/>
    <param name="set_origin_menu" type="boolean" value="true"/>
    <param name="set_origin_to_rhand_menu" type="boolean" value="true"/>
    <param name="set_origin_to_lhand_menu" type="boolean" value="true"/>
    <param name="reset_marker_pos_menu" type="boolean" value="true"/>
    <param name="manipulation_mode_menu" type="boolean" value="true"/>
    <param name="select_arm_menu" type="boolean" value="true"/>
    <param name="ik_mode_menu" type="boolean" value="true"/>
    <param name="use_torso_menu" type="boolean" value="true"/>
    <param name="publish_marker_menu" type="boolean" value="true"/>
    <param name="head_link_frame" type="string" value="head"/>
    <param name="head_mesh" type="string" value="package://multisense_sl_description/meshes/head.dae"/>
    <rosparam command="load" param="mesh_config" file="$(find jsk_interactive_marker)/launch/atlas_mesh_config.yaml"/>
    <param name="hand_type" type="string" value="sandia_hand"/>
  </node>
</launch>